crates/contracts/src/bin/vendor.rs

```diff
@@ -28,6 +28,8 @@ fn main() {
 fn run() -> Result<()> {
     let vendor = Vendor::new()?;
 
+    let ethflow_version = "0.0.0-rc.3";
```

Comment:
nit: Seems reasonable to make this an "official" constant
```suggestion
    const ETHFLOW_VERSION: &str = "0.0.0-rc.3";
```
Nice idea to make updating the contract version easier!